recipe/meta.yaml

```diff
@@ -27,13 +27,11 @@ requirements:
     - cython
     - numpy
     - libgdal
-    - geos
   run:
     - python
     - gdal
     - {{ pin_compatible('numpy') }}
     - pygeos
-    - pyproj
```

Comment:
I removed pyproj here because we only use pyproj in functions that use geopandas, and thus pyproj is available through geopandas.